spec/configuration.yml.example

```diff
@@ -7,11 +7,11 @@ root:
 user:
   host: localhost
   username: LOCALUSERNAME
-  password:
+  password: test
   database: mysql2_test
 
 numericuser:
   host: localhost
   username: LOCALUSERNAME
-  password:
+  password: test
```

Comment:
I need to check where the user is created; IIRC the assumption is that a $USER user already exists?